bluepysnap/frame_report.py

```diff
@@ -98,22 +97,36 @@ def get(self, group=None, t_start=None, t_stop=None):
         Returns:
             pandas.DataFrame: frame as columns indexed by timestamps.
         """
-        ids = [] if group is None else self._resolve(group).tolist()
-        t_start = -1 if t_start is None else t_start
-        t_stop = -1 if t_stop is None else t_stop
+        ids = self._resolve(group).tolist()
+        try:
+            view = self._frame_population.get(node_ids=ids, tstart=t_start, tstop=t_stop)
+        except SonataError as e:
+            raise BluepySnapError(e)
 
-        view = self._frame_population.get(node_ids=ids, tstart=t_start, tstop=t_stop)
-        if not view.data:
+        if len(view.ids) == 0:
             return pd.DataFrame()
-        res = pd.DataFrame(data=view.data, index=view.index)
+
+        res = pd.DataFrame(data=view.data,
+                           columns=pd.MultiIndex.from_arrays(np.asarray(view.ids).T),
+                           index=view.times).sort_index(axis=1)
```

Comment:
`view.times` are always guaranteed to be present?

Reply:
yes ! 
the view always contains the 3 objects